lib/util.js

```diff
@@ -19,9 +19,24 @@ var fs = require('fs'),
      */
     FILESIZE_OPTIONS = { spacer: '' },
 
+    POSTMAN_API_PROTOCOL = 'https',
+
     POSTMAN_API_HOST = 'api.getpostman.com',
```

Comment:
Rename this to `POSTMAN_API_URL` and change the value to `https://api.getpostman.com`